src/ui/CustomMenuUI.test.ts

```diff
@@ -704,42 +729,112 @@ describe('Custom Menu UI   ', () => {
     ).toBe(true);
   });
 
-
   it('should handle removeCustomStyleName3', () => {
     const state = {
       doc: {
         nodesBetween(from, to, callback) {
           for (let i = from; i < to; i++) {
             const node = {
               content: {
-                content: [{
-                  marks: [
-                    { attrs: { styleName: 'AFDP_Bullet' } }
-                  ]
-                }]
+                content: [
+                  {
+                    marks: [{ attrs: { styleName: 'AFDP_Bullet' } }],
+                  },
+                ],
               },
-              type:{name:'paragraph'},
-              attrs: { styleName: 'AFDP_Bullet' }
+              type: { name: 'paragraph' },
+              attrs: { styleName: 'AFDP_Bullet' },
             };
             callback(node, i);
           }
         },
-        nodeSize: 10
+        nodeSize: 10,
       },
       schema: {},
       selection: { from: 0, to: 1 },
       plugins: [],
-      empty: null ,
-      tr:{removeMark:()=>{return {setNodeMarkup:()=>{return {};},removeMark:()=>{return {setNodeMarkup:()=>{return {};},
-      removeMark:()=>{return {setNodeMarkup:()=>{return {};},removeMark:()=>{return {setNodeMarkup:()=>{return {};},
-      removeMark:()=>{return {setNodeMarkup:()=>{return {};},removeMark:()=>{return {setNodeMarkup:()=>{return {};},
-      removeMark:()=>{return {setNodeMarkup:()=>{return {};},removeMark:()=>{return {setNodeMarkup:()=>{return {};},
-      removeMark:()=>{return {setNodeMarkup:()=>{return {};},removeMark:()=>{return {setNodeMarkup:()=>{return {};}};}};}};}};}};}};}};}};}};}};}}
+      empty: null,
+      tr: {
+        removeMark: () => {
+          return {
+            setNodeMarkup: () => {
+              return {};
+            },
+            removeMark: () => {
```

Comment:
Could you not just save this object as a variable so that you can make it self-referential and avoid this manual recursion?